lib/src/subscription.dart

```diff
@@ -272,21 +272,22 @@ class MutableSubscriptionSet extends SubscriptionSet {
     return Subscription._(realmCore.insertOrAssignSubscription(this, query, name, update));
   }
 
-  /// Remove the [subscription] from the set, if it exists.
+  /// Removes the [subscription] from the set, if it exists.
   bool remove(Subscription subscription) {
     return realmCore.eraseSubscriptionById(this, subscription);
   }
 
-  /// Remove the [query] from the set, if it exists.
+  /// Removes the [query] from the set, if it exists.
   bool removeByQuery<T extends RealmObject>(RealmResults<T> query) {
     return realmCore.eraseSubscriptionByResults(this, query);
   }
 
-  /// Remove the [query] from the set that matches by [name], if it exists.
+  /// Removes the subscriptions from the set that matches by [name], if it exists.
```

Comment:
```suggestion
  /// Remove the subscription from the set that matches by [name], if it exists.
```
You can only have one with a given name